content/en/tracing/security/agent_span_modification.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Agent Span Modification
+kind: documentation
+description: "Configure the Datadog Agent to modify or discard spans for security or fine-tuning purposes."
+aliases:
+    - /tracing/guide/agent_obfuscation
+---
+
+## Filtering Baseline
+
+Several filtering mechanisms are enforced as a baseline in an effort to provide sound defaults. In particular:
+
+* **Environment variables are not collected by the Agent**
+* **SQL variables are obfuscated by default, even when not using prepared statements**: For example, the following `sql.query` attribute: `SELECT data FROM table WHERE key=123 LIMIT 10` would have its variables obfuscated, to become the following Resource name: `SELECT data FROM table WHERE key = ? LIMIT ?`
+* **Numbers in Resource names (e.g. in request urls) are obfuscated by default** For example, the following `elasticsearch` attribute:
+
+    ```text
+    Elasticsearch : {
+        method : GET,
+        url : /user.0123456789/friends/_count
+    }
+    ```
+
+    would have its number in the url obfuscated, to become the following Resource name: `GET /user.?/friends/_count`
+
+## Agent Trace Obfuscation
+
+Agent [trace][1] obfuscation is disabled by default. Enable it in your `datadog.yaml` configuration file to obfuscate all information attached to your traces.
+
+Currently this options only works with the following services:
+
+* `mongodb`
+* `elasticsearch`
+* `redis`
+* `memcached`
+* `http`
+* `remove_stack_traces`
+
+**Note:** Automatic scrubbing can be used for multiple types of services at the same time.  Each should be configured under the `obfuscation` level of your `datadog.yaml` file.
```

Comment:
```suggestion
**Note:** You can use automatic scrubbing for multiple types of services at the same time.  Configure each in the `obfuscation` section of your `datadog.yaml` file.
```

Reply:
also, you called this "obfuscation" in the heading, and "scrubbing" here. Stick with one term, or explain that they (can) mean the same thing.